x-pack/plugins/alerting/server/routes/rule/apis/types/transforms/transform_rule_types_response/v1.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RegistryAlertTypeWithAuth } from '../../../../../../authorization';
+import { TypesRulesResponseBodyV1 } from '../../../../../../../common/routes/rule/apis/rule_types';
+
+export const transformRuleTypesResponse = (
+  ruleTypes: Set<RegistryAlertTypeWithAuth>
+): TypesRulesResponseBodyV1 => {
+  return Array.from(ruleTypes).map((ruleType: RegistryAlertTypeWithAuth) => {
+    return {
+      action_groups: ruleType.actionGroups,
+      action_variables: ruleType.actionVariables,
+      alerts: ruleType.alerts,
```

Comment:
This one throws an error: Property 'alerts' does not exist on type 'RegistryAlertTypeWithAuth'.  I'll add it to the type, just let me know if you have more info/know a better option